crescent/context.py

```diff
@@ -37,31 +37,49 @@
     from crescent.bot import Bot
 
 
-__all__: Sequence[str] = ("Context",)
+__all__: Sequence[str] = ("Context", "AutocompleteContext")
 
 
 @define
-class Context:
-    """Represents the context for interactions"""
+class BaseContext:
+    """Represents the context for command interactions"""
 
     interaction: CommandInteraction
+    """The interaction object."""
     app: Bot
+    """The client application."""
```

Comment:
"client" and "application" are both nouns I think? Maybe "application instance" or "discord client" or smth.

If you think it's fine then I'm fine with it.

Reply:
I think instance makes more sense